create-ubuntu-multiplatform-image.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+set -e
+
+ARM_IMAGE="curity.azurecr.io/curity/idsvr:${VERSION}-arm"
+X86_IMAGE="curity.azurecr.io/curity/idsvr:${VERSION}-x86"
+
+docker pull "$ARM_IMAGE"
+docker pull "$X86_IMAGE"
+
+
+if [[ "$VERSION" == *.0 ]]; then
+  BRANCH_VERSION=${VERSION%??}
+  EXTRA_TAGS_UBUNTU="curity.azurecr.io/curity/idsvr:${BRANCH_VERSION}-ubuntu
+  curity.azurecr.io/curity/idsvr:${BRANCH_VERSION} curity.azurecr.io/curity/idsvr:${BRANCH_VERSION}-ubuntu18
+  curity.azurecr.io/curity/idsvr:${BRANCH_VERSION}-ubuntu18.04"
+fi
+
+# shellcheck disable=SC2206
+TAG_ARRAY=("curity.azurecr.io/curity/idsvr:${VERSION}-ubuntu18.04" "curity.azurecr.io/curity/idsvr:${VERSION}-ubuntu"
```

Comment:
one element per line would be more readable. I don't think leading whitespace would hurt either